tools/checked-c-convert/utils/run.py

```diff
@@ -0,0 +1,66 @@
+import re
+import os
+import json
+import argparse
+import traceback
+import subprocess
+
+"""
+This tool will invoke checked-c-convert on a compile_commands.json database. 
+It contains some work-arounds for cmake+nmake generated compile_commands.json 
+files, where the files are malformed. 
+
+To use, set PROG_NAME to the path where checked-c-convert.exe is built.
```

Comment:
How about making PROG_NAME an argument to the script?  This will be easier for people to use than having to edit the script.